### PR TITLE
Job links

### PR DESCRIPTION
**Pull request by zoeabryant**

### CODE REVIEW

File: shared/components/ConferenceJobs/index.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  return (
+    <div>
+      <section className="block ConferenceJobs__about">
+        <div className="content">
+          <h2>Jobs</h2>
+          <p>
+            Some Intro about Jobs Section. Lorem ipsum dolor sit amet,
+            consectetur adipiscing elit, sed do eiusmod tempor incidid
+            unt ut labore et dolore magna aliqua. Utenim ad minim veni
+            am, quis nostrud exercitation ullamco laboris nisi.
```
Comment (lpil) [inline shared/components/ConferenceJobs/index.js]: Need real copy before merging :)

**Comment by lpil** [comment]
Looks good. Is BB ready? i.e. all the job fields are in the schema on production?

**Comment by asavin** [comment]
👍

File: shared/components/ConferenceJobs/index.scss [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  }
+
+  p {
+    line-height: 1.3;
+    margin: 0px;
+    padding: 0px;
+  }
+
+  @media (min-width: $tablet-width) {
+    padding: 40px 0px 40px 0px;
```
Comment (AndrewBestbier) [inline shared/components/ConferenceJobs/index.scss]: Same as:     padding: 40px 0px;

**Comment by AndrewBestbier** [comment]
Could you make your commit messages a little clearer in future :) 'fixed the thing' is a little vague

**Comment by lpil** [comment]
Hah! I didn't notice that

File: shared/components/JobsSection/index.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+};
+
+JobPartner.propTypes = {
+  ...partnerType,
+  displayDescription: PropTypes.bool,
+};
+
+const JobsSection = ({ partners, displayDescription }) => {
+  if (isEmpty(partners)) {
+    console.log('No jobs!');
```
Comment (AndrewBestbier) [inline shared/components/JobsSection/index.js]: Rogue console.log

Comment (lpil) [inline shared/components/JobsSection/index.js]: Good catch!

File: shared/components/JobsSection/index.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    return null;
+  }
+  return (
+    <section className={'block JobsSection'}>
+      {partners.map((partner, index) =>
```
Comment (AndrewBestbier) [inline shared/components/JobsSection/index.js]: This feels odd, maybe put the } on the a new line to make it look cleaner

**Comment by AndrewBestbier** [comment]
Tiny comments, then +1 :)

File: shared/components/JobsSection/index.js (hunk 4 above)
Comment (lpil) [inline shared/components/JobsSection/index.js]: We're doing this in other places in the app :)

Comment (AndrewBestbier) [inline shared/components/JobsSection/index.js]: Fair enough. As you were

**Comment by zoeabryant** [comment]
Badger brain is set up @lpil 
Noted @AndrewBestbier re commit messages 👍

**Comment by AndrewBestbier** [comment]
@zoeabryant why is this one blocked again? :)

**Comment by lpil** [comment]
@AndrewBestbier It has lorem ipsum in it

File: shared/components/ConferenceJobs/index.js [hunk 5]
```diff
@@ -4,7 +4,7 @@ import JobsSection from '../JobsSection';
 
 const isEmpty = array => !Array.isArray(array) || array.length === 0;
 
-const hasNoJobs = array => array.every(partner => isEmpty(partner.jobs));
+const hasNoJobs = array => Array.isArray(array) && array.every(partner => isEmpty(partner.jobs));
```
Comment (lpil) [inline shared/components/ConferenceJobs/index.js]: @zoeabryant This will return false if the array is undefined. It should be true.

**Pull request merged**